dataPrepII_PCC/makePCCRateFile.py

```diff
@@ -193,12 +193,15 @@ def extractPCCRates(clusters, PCC_BCID, in_data):
 def doMakePCCRateFile(ConfigInfo):
     
     InputPCCFiles = ConfigInfo['InputPCCFiles']
-    AnalysisDir = ConfigInfo['AnalysisDir']
+    AnalysisDir   = ConfigInfo['AnalysisDir']
     InputScanFile = AnalysisDir + "/" + ConfigInfo['InputScanFile']
-    PCCTreeName = ConfigInfo['PCCTreeName']
-    PCC_BCID = ConfigInfo['PCC_BCID']
-    addScanInfo = False
-    addScanInfo = ConfigInfo['addScanInfo']
+    PCCTreeName   = ConfigInfo['PCCTreeName']
+    PCC_BCID      = ConfigInfo['PCC_BCID']
+    addScanInfo   = False
+    addScanInfo   = ConfigInfo['addScanInfo']
+    DataType      = ConfigInfo['DataType']
+
+    print DataType
```

Comment:
I mean the line 'print DataType'. Widthwise, if it a line that gives info to the user, please change it to 
print 'DataType to processed',DataType